x-pack/legacy/plugins/actions/server/routes/fire.test.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+jest.mock('../lib/execute', () => ({
+  execute: jest.fn(),
+}));
+
+import { createMockServer } from './_mock_server';
+import { fireRoute } from './fire';
+
+const getServices = jest.fn();
+
+const { server, actionTypeRegistry, savedObjectsClient } = createMockServer();
+fireRoute({ server, actionTypeRegistry, getServices });
+
+beforeEach(() => jest.resetAllMocks());
+
+it('fires an action with proper parameters', async () => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const { execute } = require('../lib/execute');
+
+  const request = {
+    method: 'POST',
+    url: '/api/action/1/_fire',
```

Comment:
Still not loving `fire` for actions.  Thinking this should be execute maybe.

Reply:
I went ahead and created a To-Do in the description here https://github.com/elastic/kibana/issues/24214.